explorer/idl/explorer.idl

```diff
@@ -408,7 +408,7 @@ interface Explorer {
     getPeers() GetPeersResponse
 
     // get receipt by execution id
-    getReceiptByExecutionID(id string) Receipt
+    getReceiptByActionID(id string) Receipt
```

Comment:
Instead of modifying the endpoint, it's safer to keep it and create a new endpoint cc @puncsky

Reply:
I keep the original endpoint and make "getReceiptByActionID" a new one.